engine/samples/voxels/page.md

```diff
@@ -0,0 +1,30 @@
+# Voxels {#examples-engine-voxels}
+
+@brief Using the @ref voxels-plugin plugin.
+
+This example shows how the @ref voxels-plugin plugin, can be used to load voxel grids.
+
+![](voxels/output.png)
+
+The plugin function is included from the @ref engine/voxels/plugin.hpp header.
+To see the asset, you'll also need to perform a basic configuration of the @ref renderer-plugin plugin like shown in the @ref examples-renderer-sample sample.
+
+@snippet voxels/main.cpp Adding the plugin
+
+Lets start by getting the assets we want to use. Each asset has an id through which we can get a handle to it. This id is specified in the .meta file associated to the asset.
+
+@snippet voxels/main.cpp Get handles to assets
+
+Next, let's assign the palette, that we have in our assets folder, to be used by the renderer.
```

Comment:
```suggestion
In this sample, instead of creating a new palette, we just read the data from the asset identified from the `PaletteAsset` handle we defined previously. Internally, the @ref cubos::engine::Assets "assets manager" will automatically load the palette asset if it hasn't been loaded before.
```